src/test/java/pl/wedding/management/service/reservation/domain/converter/ReservationConverterTest.java

```diff
@@ -0,0 +1,134 @@
+package pl.wedding.management.service.reservation.domain.converter;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import pl.wedding.management.service.reservation.domain.entity.ReservationDto;
+import pl.wedding.management.service.reservation.domain.entity.ReservationEntity;
+import pl.wedding.management.service.reservation.domain.enums.Occasion;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ReservationConverterTest {
+
+    ReservationConverter reservationConverter;
+    ReservationDto reservationDto;
+
+    @BeforeEach
+    void setUp() {
+        reservationConverter = new ReservationConverter();
+        reservationDto = new ReservationDto(
+                1,
+                LocalDateTime.of(2020,Month.OCTOBER,3,18,30,0),
+                LocalDateTime.of(2020,Month.OCTOBER,4,6,30,0),
+                new Object(),
+                new Object(),
+                200,
+                Occasion.DIVORCE_PARTY
+        );
+    }
+
+    @Test
+    void convertFromEntityIsEqual() {
+
+        //given
+        ReservationEntity reservationEntity = new ReservationEntity(
+                1,
+                LocalDateTime.of(2020,Month.OCTOBER,3,18,30,0),
+                LocalDateTime.of(2020,Month.OCTOBER,4,6,30,0),
+                new Object(),
+                new Object(),
+                200,
+                Occasion.DIVORCE_PARTY
+        );
+
+        //when
+        ReservationDto reservationDtoFromConverter = reservationConverter.convertFromEntity(reservationEntity);
```

Comment:
Nazwij tą zmienną jako "result". To jest dobra praktyka nazywania zmiennych, które są wynikiem testowanej metody.